core/trino-main/src/test/java/io/trino/sql/planner/planprinter/TestWindowOperatorStats.java

```diff
@@ -33,7 +33,7 @@ public void testEmptyDriverInfosList()
         assertThat(stats.getIndexCountPerDriverStdDev()).isNaN();
         assertThat(stats.getPartitionRowsStdDev()).isNaN();
         assertThat(stats.getRowsPerDriverStdDev()).isNaN();
-        assertThat(stats.getActiveDrivers()).isEqualTo(0);
-        assertThat(stats.getTotalDrivers()).isEqualTo(0);
+        assertThat(stats.getActiveDrivers()).isZero();
+        assertThat(stats.getTotalDrivers()).isZero();
```

Comment:
i prefer the previous version, but whatever.... they shouldn't have added `isZero` in the first place.